omise/omise-magento
# Delete Saved Credit Card Information

#### 1. Objective

This Pull Request deliver functionality to delete saved credit card.

#### 2. Description of change

This PR adds to the customer profile new tab called "Omise Payment Methods".

#### 3. Quality assurance

**🔧 Environments:**

- **Platform version**: Magento CE 2.2.4.
- **Omise plugin version**: Omise-Magento 2.7.
- **PHP version**: 7.0.29.

**✏️ Details:**

To manually test this feature:

*When no card is saved*
- Login to your magento 2 customer profile
- Open customer profile and check if on tab _Omise Payment Methods_ you have information that no payment card is available.
![Screenshot 2019-03-28 at 18 24 09](https://user-images.githubusercontent.com/10651523/55154193-c55a2e80-5186-11e9-8301-0e4f0125c4cc.png)

*When you have saved credit card*
- Make sure you are logged in as customer
- Check if in _Omise Payment Methods_ you have listed all saved credit cards.
- Delete credit card.
- Make sure deleted card is no more listed on the list.

#### 4. Impact of the change
New Tab was added to Magento 2 Customer Profile.

![Screenshot 2019-03-28 at 17 44 25](https://user-images.githubusercontent.com/10651523/55153826-fb4ae300-5185-11e9-94e0-bd1af2c49f94.png)
